build-image/first-boot.bash

```diff
@@ -22,14 +18,20 @@ fail_inprogress() {
 echo "$(timestamp) [openHABian] Starting the openHABian initial setup."
 rm -f /opt/openHABian-install-failed
 touch /opt/openHABian-install-inprogress
-
 echo -n "$(timestamp) [openHABian] Storing configuration... "
 cp /boot/openhabian.conf /etc/openhabian.conf
 sed -i 's/\r$//' /etc/openhabian.conf
-# shellcheck disable=SC1091
-source /etc/openhabian.conf
+
+source "init.bash"
```

Comment:
Sorry, I have overlooked this. We have a problem here:
You cannot just source it, it will only work if current directory matches the one of the script. It might be the case for first-boot.bash, but not in genreal.
Looking at current init.bash - isn't it easier to drop it and use the hard coded filename instead of $CONFIGFILE?

Reply:
I added configfile= to /etc/openhabian.conf because I dislike hardcoding strings